presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/aggregation/TestGeometryStateSerializer.java

```diff
@@ -28,11 +30,19 @@
 
 public class TestGeometryStateSerializer
 {
+    private AccumulatorStateFactory<GeometryState> factory;
+    private AccumulatorStateSerializer<GeometryState> serializer;
+
+    @BeforeMethod
+    public void init()
```

Comment:
we usually call this method  `setUp()` or  `setUpX` . Let us stick to that standard.

Reply:
Done.